src/api/sakApi.ts

```diff
@@ -27,8 +27,12 @@ export async function fetchSakBySakId(sakId: string): Promise<ApiClientResult<Sa
     return apiClient({ url: `/saker/${sakId}`, method: 'GET' });
 }
 
-export async function hentRestanser(): Promise<ApiClientResult<Restans[]>> {
-    return apiClient({ url: `/saker/restanser`, method: 'GET' });
+export async function hentÅpneBehandlinger(): Promise<ApiClientResult<Restans[]>> {
+    return apiClient({ url: `/saker/apneBehandlinger`, method: 'GET' });
+}
+
+export async function hentFerdigeBehandlinger(): Promise<ApiClientResult<Restans[]>> {
+    return apiClient({ url: `/saker/ferdigeBehandlinger`, method: 'GET' });
```

Comment:
Er det noen sp. grunn till att man har valgt å ha to olika endepunkter for åpne og ferdige behandlinger? alternativet hade ju varit å bara hentet alla behandlinger på startsiden, og så blir det filtrert i tabsen f.eks. 😊

Reply:
Vi diskuterte det litt, og tenkte at det var mest REST å dele det i to. I tillegg kan det tenkes at den spørringen kan bli litt stor, etter hvert som tiden går. _I tillegg_ er det litt unødvendig å bruke tid på å laste data man ikke trenger (dersom man ikke trykker på `ferdige behandlinger` feks) 😄